jdk/test/java/io/File/createTempFile/SpecialTempFile.java

```diff
@@ -21,6 +21,12 @@
  * questions.
  */
 
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2022, 2022 All Rights Reserved
+ * ===========================================================================
+ */
+
```

Comment:
I don't think we want to add this, or did I miss some changes that weren't just back-ported?

Reply:
I had to modify `OSVersion.current()` to `System.getProperty("os.version")`

Reply:
Ok, but then it should say 2025, not 2022.

Reply:
Ya, updated.